forge-gui/res/cardsfolder/upcoming/become_anonymous.txt

```diff
@@ -1,8 +1,9 @@
 Name:Become Anonymous
 ManaCost:2 U U
 Types:Instant
-A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Creature.nonToken+YouOwn | TgtPrompt$ Select target nontoken creature you own | RememberChanged$ True | SubAbility$ DBExileTopCard | SpellDescription$ Exile target nontoken creature you own and the top two cards of your library in a face-down pile, shuffle that pile, then cloak those cards. They enter the battlefield tapped. (To cloak a card, put it onto the battlefield face down as a 2/2 creature with ward 2. Turn it face up any time for its mana cost if it’s a creature card.)
-SVar:DBExileTopCard:DB$ Dig | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | ExileFaceDown$ True | SubAbility$ DBCloak
+A:SP$ Pump | Origin$ Battlefield | ValidTgts$ Creature.nonToken+YouOwn | TgtPrompt$ Select target nontoken creature you own | RememberTargets$ True | SubAbility$ DBExileTopCard | SpellDescription$ Exile target nontoken creature you own and the top two cards of your library in a face-down pile, shuffle that pile, then cloak those cards. They enter the battlefield tapped. (To cloak a card, put it onto the battlefield face down as a 2/2 creature with ward {2}. Turn it face up any time for its mana cost if it's a creature card.)
+SVar:DBExileTopCard:DB$ Dig | Defined$ You | DigNum$ 2 | ChangeNum$ All | Reveal$ True | NoMove$ True | RememberRevealed$ True | SubAbility$ ExileAll
```

Comment:
why reveal?

Reply:
does it even work now...?

Reply:
With noreveal it does seem to work

Reply:
are you sure you don't get to see your library cards?

Reply:
No more reveal this way